### PR TITLE
S02 footer about us

### PR DESCRIPTION
**Pull request by FGAT3**

### CODE REVIEW

**Review by pjroggy — APPROVED** [review]

**Review by JadeJolivet — APPROVED** [review]

File: client/src/App.jsx
Comment (Yoann-Fortin) [inline client/src/App.jsx:1]: A dedicated file would be better

File: client/src/components/GameListModal/GameListModal.jsx
Comment (Yoann-Fortin) [inline client/src/components/GameListModal/GameListModal.jsx:1]: Props cannot be both optional and required

File: client/src/components/NavigationChevron/NavigationChevron.jsx
Comment (Yoann-Fortin) [inline client/src/components/NavigationChevron/NavigationChevron.jsx:1]: Separates logic from semantics

File: client/src/pages/GameListPage/GameListPage.jsx
Comment (Yoann-Fortin) [inline client/src/pages/GameListPage/GameListPage.jsx:1]: Fill in only the necessary props

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: client/src/components/ChevronButton/ChevronButton.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import PropTypes from "prop-types";
+import "./ChevronButton.scss";
+
+function ChevronButton({ onClick, src, ariaLabel }) {
+  return (
+    <button
+      type="button"
+      aria-label={ariaLabel}
```
Comment (Yoann-Fortin) [inline client/src/components/ChevronButton/ChevronButton.jsx]: If you have a decorative image, take advantage of the opportunity to specify the button's function in the alt attribute of the image contained in the button.

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**